client/src/Block/components/BlockDetailsCard.tsx

```diff
@@ -0,0 +1,96 @@
+import { Block } from "gql/graphql";
+import { FC } from "react";
+
+type Props = {
+  block: Block;
+};
+
+const BlockDetailsCard: FC<Props> = ({ block }) => {
+  return (
+    <div className="w-full">
+      <div className="flex items-center justify-between mb-10">
+        <h3 className="font-semibold leading-none text-gray-900 text-2xl">
+          Block #{block.height}
+        </h3>
+      </div>
+      <div className="border border-slate-100 bg-white shadow rounded-lg mb-4 p-4 sm:p-6 w-full">
+        <div className="flow-root">
+          <ul className="divide-y divide-gray-200">
+            <li className="py-3 sm:py-4">
```

Comment:
Very minor, feel free to ignore, we can reduce duplication by extracting some components here